client/components/with-poll-base/index.js

```diff
@@ -21,7 +21,7 @@ import usePollDuplicateCleaner from 'components/use-poll-duplicate-cleaner';
 
 startSubscriptions();
 
-const isP2tenberg = () => 'p2tenberg' in window;
+const isP2tenberg = () => 'p2tenberg' in window || 'p2editor' in window;
```

Comment:
Is this supposed to be in this patch?

Reply:
Yeah, it's that old issue/suggestion from @johngodley I thought I could throw it here and not make another PR for it, but happy to decouple it if it seems messy